packages/nova-react-test-utils/package.json

```diff
@@ -11,35 +11,49 @@
     "just": "monorepo-scripts"
   },
   "peerDependencies": {
```

Comment:
Should we use `https://docs.npmjs.com/cli/v8/configuring-npm/package-json#peerdependenciesmeta` to identify some of these are "optional" depending on the variant you use?